Thu Jul  3 14:49:36 2025       
+-----------------------------------------------------------------------------------------+
| NVIDIA-SMI 550.163.01             Driver Version: 550.163.01     CUDA Version: 12.4     |
|-----------------------------------------+------------------------+----------------------+
| GPU  Name                 Persistence-M | Bus-Id          Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |           Memory-Usage | GPU-Util  Compute M. |
|                                         |                        |               MIG M. |
|=========================================+========================+======================|
|   0  NVIDIA A100-SXM4-80GB          On  |   00000000:03:00.0 Off |                    0 |
| N/A   28C    P0             62W /  500W |       1MiB /  81920MiB |      0%      Default |
|                                         |                        |             Disabled |
+-----------------------------------------+------------------------+----------------------+
|   1  NVIDIA A100-SXM4-80GB          On  |   00000000:41:00.0 Off |                    0 |
| N/A   28C    P0             62W /  500W |       1MiB /  81920MiB |      0%      Default |
|                                         |                        |             Disabled |
+-----------------------------------------+------------------------+----------------------+
|   2  NVIDIA A100-SXM4-80GB          On  |   00000000:82:00.0 Off |                    0 |
| N/A   29C    P0             60W /  500W |       1MiB /  81920MiB |      0%      Default |
|                                         |                        |             Disabled |
+-----------------------------------------+------------------------+----------------------+
|   3  NVIDIA A100-SXM4-80GB          On  |   00000000:C1:00.0 Off |                    0 |
| N/A   27C    P0             63W /  500W |       1MiB /  81920MiB |      0%      Default |
|                                         |                        |             Disabled |
+-----------------------------------------+------------------------+----------------------+
                                                                                         
+-----------------------------------------------------------------------------------------+
| Processes:                                                                              |
|  GPU   GI   CI        PID   Type   Process name                              GPU Memory |
|        ID   ID                                                               Usage      |
|=========================================================================================|
|  No running processes found                                                             |
+-----------------------------------------------------------------------------------------+
W0703 14:49:49.587000 140077208918912 torch/distributed/run.py:779] 
W0703 14:49:49.587000 140077208918912 torch/distributed/run.py:779] *****************************************
W0703 14:49:49.587000 140077208918912 torch/distributed/run.py:779] Setting OMP_NUM_THREADS environment variable for each process to be 1 in default, to avoid your system being overloaded, please further tune the variable for optimal performance in your application as needed. 
W0703 14:49:49.587000 140077208918912 torch/distributed/run.py:779] *****************************************
Configurator found file config/model_6_1_1.json.
Configurator found file config/model_6_1_1.json.
Configurator found file config/model_6_1_1.json.
Configurator found file config/model_6_1_1.json.
Traceback (most recent call last):
  File "/pscratch/sd/e/eshy/particleGPT/train_new.py", line 33, in <module>
Traceback (most recent call last):
  File "/pscratch/sd/e/eshy/particleGPT/train_new.py", line 33, in <module>
    import model
  File "/pscratch/sd/e/eshy/particleGPT/model.py", line 22, in <module>
    import model
Traceback (most recent call last):
  File "/pscratch/sd/e/eshy/particleGPT/train_new.py", line 33, in <module>
Traceback (most recent call last):
  File "/pscratch/sd/e/eshy/particleGPT/model.py", line 22, in <module>
  File "/pscratch/sd/e/eshy/particleGPT/train_new.py", line 33, in <module>
    import model
    import model
  File "/pscratch/sd/e/eshy/particleGPT/model.py", line 22, in <module>
  File "/pscratch/sd/e/eshy/particleGPT/model.py", line 22, in <module>
            from dictionary import Dictionaryfrom dictionary import Dictionaryfrom dictionary import Dictionary


  File "/pscratch/sd/e/eshy/particleGPT/dictionary.py", line 399, in <module>
  File "/pscratch/sd/e/eshy/particleGPT/dictionary.py", line 399, in <module>
  File "/pscratch/sd/e/eshy/particleGPT/dictionary.py", line 399, in <module>
    from dictionary import Dictionary
  File "/pscratch/sd/e/eshy/particleGPT/dictionary.py", line 399, in <module>
        dict = Dictionary(script_dir / 'data' / 'preparation_5_10' / 'dictionary.json')dict = Dictionary(script_dir / 'data' / 'preparation_5_10' / 'dictionary.json')

    dict = Dictionary(script_dir / 'data' / 'preparation_5_10' / 'dictionary.json')
                           ^  ^^^ ^^ ^^^ ^^ ^^^^^^^    ^^^dict = Dictionary(script_dir / 'data' / 'preparation_5_10' / 'dictionary.json')^^^^
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ^^^ ^^^^^ ^^ ^^^ ^^^ ^^^^^ ^^^^ ^^ ^^^^ ^^ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
^^^^^^^^^^^^  File "/pscratch/sd/e/eshy/particleGPT/dictionary.py", line 178, in __init__

^^^^^^^^  File "/pscratch/sd/e/eshy/particleGPT/dictionary.py", line 178, in __init__
^^^^^^^^^^^^^^^^^^^^
^^^^^^  File "/pscratch/sd/e/eshy/particleGPT/dictionary.py", line 178, in __init__
^^^^^^^^^^^^^^^^^^^
  File "/pscratch/sd/e/eshy/particleGPT/dictionary.py", line 178, in __init__
    self.pt_bins    = self._create_bins('pt')
        self.pt_bins    = self._create_bins('pt')self.pt_bins    = self._create_bins('pt')

                                                         ^   ^  ^  ^ ^^     ^^self.pt_bins    = self._create_bins('pt')^^^
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ^^^ ^^^ ^^^ ^^^ ^
^ ^^ ^^ 
 ^ 
   File "/pscratch/sd/e/eshy/particleGPT/dictionary.py", line 246, in _create_bins
       File "/pscratch/sd/e/eshy/particleGPT/dictionary.py", line 246, in _create_bins
   File "/pscratch/sd/e/eshy/particleGPT/dictionary.py", line 246, in _create_bins
     ^^^^^^^^^^^^^^^^^^^^^^^
  File "/pscratch/sd/e/eshy/particleGPT/dictionary.py", line 246, in _create_bins
    all_values = get_all_of(self, self, type_str)
    all_values = get_all_of(self, self, type_str)
    all_values = get_all_of(self, self, type_str)
             all_values = get_all_of(self, self, type_str)
                                    ^  ^   ^  ^^ ^^ ^^^^ ^^^ ^^ ^^^^^^ ^^ ^^^^ ^ ^^^ ^^ ^^^^ ^^^ ^^^ ^^^ ^^^^ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
^^^^^^^  File "/pscratch/sd/e/eshy/particleGPT/dictionary.py", line 145, in get_all_of
^^

^^^^^  File "/pscratch/sd/e/eshy/particleGPT/dictionary.py", line 145, in get_all_of
  File "/pscratch/sd/e/eshy/particleGPT/dictionary.py", line 145, in get_all_of
^^^^^^^^^^^^
  File "/pscratch/sd/e/eshy/particleGPT/dictionary.py", line 145, in get_all_of
    in_dataset = dm.load_geant4_dataset(input_data_filepath, flattened_events=False, pad_token=np.nan)
    in_dataset = dm.load_geant4_dataset(input_data_filepath, flattened_events=False, pad_token=np.nan)
    in_dataset = dm.load_geant4_dataset(input_data_filepath, flattened_events=False, pad_token=np.nan)
     in_dataset = dm.load_geant4_dataset(input_data_filepath, flattened_events=False, pad_token=np.nan) 
                            ^  ^ ^  ^    ^   ^   ^   ^   ^    ^ ^^ ^^  ^^  ^^^ ^^ ^ ^^ ^^^ ^^^ ^^^ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
^^^^^^^^^^  File "/pscratch/sd/e/eshy/particleGPT/data_manager.py", line 11, in load_geant4_dataset

^^^^^^  File "/pscratch/sd/e/eshy/particleGPT/data_manager.py", line 11, in load_geant4_dataset
^^^^^^^^^^^^^^^^^
^^^^
  File "/pscratch/sd/e/eshy/particleGPT/data_manager.py", line 11, in load_geant4_dataset
  File "/pscratch/sd/e/eshy/particleGPT/data_manager.py", line 11, in load_geant4_dataset
    if not dataset_filepath.exists():
    if not dataset_filepath.exists():
     if not dataset_filepath.exists():  
              ^ ^  ^ ^ ^^^^^ ^^ ^^ ^^^^^^^^^^^^^       ^ ^^^^^^^^^^^^^^^^^^^^
^^^^^AttributeError^^^: ^
'Dictionary' object has no attribute 'exists'^AttributeError^^^^^^: ^'Dictionary' object has no attribute 'exists'^^^^^
AttributeError:     'Dictionary' object has no attribute 'exists'if not dataset_filepath.exists():


           ^^^^^
^^^^^^^^^^^^^^^^^^
AttributeError: 'Dictionary' object has no attribute 'exists'
W0703 14:50:04.409000 140077208918912 torch/distributed/elastic/multiprocessing/api.py:858] Sending process 2228343 closing signal SIGTERM
E0703 14:50:04.441000 140077208918912 torch/distributed/elastic/multiprocessing/api.py:833] failed (exitcode: 1) local_rank: 1 (pid: 2228344) of binary: /global/common/software/nersc9/pytorch/2.3.1/bin/python3.11
Traceback (most recent call last):
  File "/global/homes/e/eshy/.local/bin/torchrun", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/global/homes/e/eshy/.local/lib/python3.11/site-packages/torch/distributed/elastic/multiprocessing/errors/__init__.py", line 348, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/global/homes/e/eshy/.local/lib/python3.11/site-packages/torch/distributed/run.py", line 901, in main
    run(args)
  File "/global/homes/e/eshy/.local/lib/python3.11/site-packages/torch/distributed/run.py", line 892, in run
    elastic_launch(
  File "/global/homes/e/eshy/.local/lib/python3.11/site-packages/torch/distributed/launcher/api.py", line 133, in __call__
    return launch_agent(self._config, self._entrypoint, list(args))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/global/homes/e/eshy/.local/lib/python3.11/site-packages/torch/distributed/launcher/api.py", line 264, in launch_agent
    raise ChildFailedError(
torch.distributed.elastic.multiprocessing.errors.ChildFailedError: 
============================================================
train_new.py FAILED
------------------------------------------------------------
Failures:
[1]:
  time      : 2025-07-03_14:50:04
  host      : nid008605-hsn0
  rank      : 2 (local_rank: 2)
  exitcode  : 1 (pid: 2228345)
  error_file: <N/A>
  traceback : To enable traceback see: https://pytorch.org/docs/stable/elastic/errors.html
[2]:
  time      : 2025-07-03_14:50:04
  host      : nid008605-hsn0
  rank      : 3 (local_rank: 3)
  exitcode  : 1 (pid: 2228346)
  error_file: <N/A>
  traceback : To enable traceback see: https://pytorch.org/docs/stable/elastic/errors.html
------------------------------------------------------------
Root Cause (first observed failure):
[0]:
  time      : 2025-07-03_14:50:04
  host      : nid008605-hsn0
  rank      : 1 (local_rank: 1)
  exitcode  : 1 (pid: 2228344)
  error_file: <N/A>
  traceback : To enable traceback see: https://pytorch.org/docs/stable/elastic/errors.html
============================================================
srun: error: nid008605: task 0: Exited with exit code 1
srun: Terminating StepId=40095646.0
